Scripts/script-MD2HTML.yml

```diff
@@ -0,0 +1,29 @@
+commonfields:
+  id: MD2HTML
+  version: -1
+name: MD2HTML
+script: |-
+  from markdown2 import Markdown
+
+  markdowner = Markdown(extras=["tables"])
+  file_entry = demisto.args().get('file_id', '')
+  file_path = demisto.getFilePath(file_entry)['path']
+  demisto.results(file_path)
+  fopen = open(str(file_path), "r")
+  data = fopen.read()
+  res = markdowner.convert(data)
+  demisto.results(fileResult("md2htmlresult.html", res))
```

Comment:
Might want to rethink if you want the file name to be static like that, you should have access to the original file name, not sure.